components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,86 @@
+@import "@not-govuk/sass-base";
+@import "govuk-frontend/govuk/helpers/_colour.scss";
+
+.hods-timeline {
+  overflow: hidden;
+  position: relative;
+
+  p {
+    text-decoration: inherit;
+    font-size: inherit;
+  }
```

Comment:
This isn't right. This component has no business re-defining the styles for the `<p>` tag across the board.

Reply:
Without this stying, the bottom event doesn't display consistently. The font size is larger than the others, even though they are all set to `govuk-font(16);`
However, I mentioned in another comment threat that changing the number doesnt change the font size. This is likely why. Changing the item from a ReactNode to text fixed this problem, but it means going back to using unformatted text only.

Reply:
In that case, we might need to fix it elsewhere.